tests/modeladmin/tests.py

```diff
@@ -812,6 +813,33 @@ class BandAdmin(ModelAdmin):
             ["extra", "transport", "id", "DELETE", "main_band"],
         )
 
+    def test_multiple_log_actions(self):
+        """
+        Test for the newly introduced ModelAdmin.log_deletions()
+        and the LogEntryManager.log_actions()
+
+        """
+        ma = ModelAdmin(Band, self.site)
+        mock_request = MockRequest()
+        mock_request.user = User.objects.create(username="akash")
+        content_type = get_content_type_for_model(self.band)
+
+        flag = DELETION
+        band1 = Band.objects.create(name="The Beatles",
+                                    bio="A legendary rock band from Liverpool.",
+                                    sign_date=date(1962, 1, 1))
+        band2 = Band.objects.create(name="Mohiner Ghoraguli",
+                                    bio="A progressive rock band from Calcutta.",
+                                    sign_date=date(1967, 1, 1))
+        queryset = Band.objects.filter(Q(pk=band1.pk) | Q(pk=band2.pk))
+        created = ma.log_deletions(mock_request, queryset)
+        fetched = LogEntry.objects.filter(action_flag=flag).order_by('-id')[:2]
+        self.assertEqual(len(created), len(fetched))
+        for item in fetched:
+            self.assertEqual(item.action_flag, flag)
+            self.assertEqual(item.content_type, content_type)
+            self.assertEqual(item.user, mock_request.user)
```

Comment:
There are gaps in the assertions here: only asserting for length then common values isn't enough. Also the limit isn't necessary (the order by is good as result ordering isn't guaranteed to be deterministic 👍)

Something like this would be more complete. (I didn't run the test & please blackify as necessary):

```suggestion
        band1 = Band.objects.create(name="The Beatles",
                                    bio="A legendary rock band from Liverpool.",
                                    sign_date=date(1962, 1, 1))
        band2 = Band.objects.create(name="Mohiner Ghoraguli",
                                    bio="A progressive rock band from Calcutta.",
                                    sign_date=date(1967, 1, 1))
        queryset = Band.objects.all()
        created = ma.log_deletions(mock_request, queryset)
        logs = LogEntry.objects.all()
        assert(set(created.values()), set(logs.values()))
        log_values = list(logs.order_by("-id").values_list(
            "user",
            "content_type",
            "object_id",
            "object_repr",
            "action_flag",
            "change_message",
        ))
        expected_log_values = [
            (mock_request.user, content_type, band2.pk, repr(band2), flag, ""),
            (mock_request.user, content_type, band1.pk, repr(band1), flag, ""),
        ]
        self.assertEqual(log_values, expected_log_values)
```